Fleece/PlatformCompat.cc

```diff
@@ -0,0 +1,48 @@
+#ifdef WIN32
```

Comment:
The header is testing `_MSC_VER`, but here you test `WIN32`. Shouldn't they test the same thing? Otherwise if somehow we build with MSVC but not on Windows, the build will break (or something else bad will happen.)

Are there other places where we should be checking `WIN32` instead of `_MSC_VER`?